libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/Telemetry.kt

```diff
@@ -39,24 +37,12 @@ object Telemetry {
             },
         )
     ).execute()
-    return when (val e = response.exception) {
-      null -> {
-        Result.success(Unit)
-      }
 
-      is ApolloHttpException -> {
-        val body = e.body?.use { it.readUtf8() } ?: ""
-        Result.failure(Exception("Cannot track Apollo Kotlin usage: (code: ${e.statusCode})\n$body", e))
-      }
-
-      is ApolloGraphQLException -> {
-        Result.failure(Exception("Cannot track Apollo Kotlin usage: ${e.errors.joinToString { it.message }}"))
-      }
-
-      else -> {
-        Result.failure(Exception("Cannot track Apollo Kotlin usage: ${e.message}", e))
-      }
+    if (response.data == null) {
+      throw response.toException("Cannot track Apollo Kotlin usage")
     }
+
+    return Result.success(Unit)
```

Comment:
Should we check `data.trackApolloKotlinUsage` for a specific value here?

Reply:
`trackApolloKotlinUsage` is always `null` (the `Void` scalar)

Reply:
I mapped `Void` to `Unit` [here](https://github.com/apollographql/apollo-kotlin/pull/5452/commits/edd88ade9a1a08c1aa9f68e915f72b007ded1383).

Funnily, `Void` can be nullable so it's a `null` of `null` 😄

Reply:
Unless `Void` is an error 👀

Reply:
>Funnily, Void can be nullable so it's a null of null 😄

🤯